drivers/soundwire/intel_auxdevice.c

```diff
@@ -275,6 +275,8 @@ int intel_link_startup(struct auxiliary_device *auxdev)
 
```

Comment:
Should we use `Closes:` instead of `Link:` in commit message?

Reply:
ah yes, I started working on this before the Closes stuff was added. Thanks for the feedback @bardliao